pyproject.toml

```diff
@@ -14,7 +14,7 @@ description = "Hydrological Simulation Program - Python"
 dependencies = [
     "cltoolbox",
     "numba",
-    "pandas",
+    "pandas>=1.5,<2",    
```

Comment:
we should refactor all calls to `df.append()` since it's deprecated. use `pandas.concat([df1, df2]) going forward`